recipes-debian/ncurses/ncurses_debian.bbappend

```diff
@@ -0,0 +1 @@
+CVE_VERSION = "${@d.getVar("PV").split('+')[0]}"
```

Comment:
Isn't "-20181013" necessary instead of "+20181013"?

https://nvd.nist.gov/vuln/detail/CVE-2019-17595

Reply:
I found the following expression. So should we use "6.1.20181013" style?

https://nvd.nist.gov/vuln/detail/CVE-2019-17595/cpes?expandCpeRanges=true#range-4392788
> cpe:2.3:a:gnu:ncurses:6.1.20180414:*:*:*:*:*:*:*

Reply:
I guess "6.1" is better than "6.1.20181013" style.
There are some CVEs that don't have date in version.
https://nvd.nist.gov/vuln/detail/CVE-2018-19211

I think version "6.1" means including all version of 6.1.
If use "6.1.20181013" style, CVE-2018-19211 is judged as Patched.

Reply:
Okay, we should use "6.1" style because the CPE may use a coarse version.

Reply:
On the other hand, the CVE-2018-19211 is fixed in 6.1-20180414. So we need to add it to the CVE_CHECK_WHITELIST after your change is merged.
https://bugzilla.redhat.com/show_bug.cgi?id=1643754#c8